user.go

```diff
@@ -9,20 +9,20 @@ import (
 
 // User describes a user account.
 type User struct {
-	ID            string         `json:"id,omitempty"`
-	Email         string         `json:"email,omitempty"`
-	FirstName     string         `json:"first_name,omitempty"`
-	LastName      string         `json:"last_name,omitempty"`
-	Username      string         `json:"username,omitempty"`
-	Telephone     string         `json:"telephone,omitempty"`
-	Country       string         `json:"country,omitempty"`
-	Zipcode       string         `json:"zipcode,omitempty"`
-	CreatedOn     *time.Time     `json:"created_on,omitempty"`
-	ModifiedOn    *time.Time     `json:"modified_on,omitempty"`
-	APIKey        string         `json:"api_key,omitempty"`
-	TwoFA         bool           `json:"two_factor_authentication_enabled,omitempty"`
-	Betas         []string       `json:"betas,omitempty"`
-	Organizations []Organization `json:"organizations,omitempty"`
+	ID         string     `json:"id,omitempty"`
+	Email      string     `json:"email,omitempty"`
+	FirstName  string     `json:"first_name,omitempty"`
+	LastName   string     `json:"last_name,omitempty"`
+	Username   string     `json:"username,omitempty"`
+	Telephone  string     `json:"telephone,omitempty"`
+	Country    string     `json:"country,omitempty"`
+	Zipcode    string     `json:"zipcode,omitempty"`
+	CreatedOn  *time.Time `json:"created_on,omitempty"`
+	ModifiedOn *time.Time `json:"modified_on,omitempty"`
+	APIKey     string     `json:"api_key,omitempty"`
+	TwoFA      bool       `json:"two_factor_authentication_enabled,omitempty"`
+	Betas      []string   `json:"betas,omitempty"`
+	Accounts   []Account  `json:"organizations,omitempty"`
```

Comment:
I've updated the field name to be correct however the JSON response is still for `organizations` so this needs to stay until the API responses are either updated or conditionally respond with different values based on the incoming request.